components/org.wso2.carbon.identity.workflow.engine/src/main/java/org.wso2.carbon.identity.workflow.engine/DefaultWorkflowEventRequestService.java

```diff
@@ -0,0 +1,151 @@
+package org.wso2.carbon.identity.workflow.engine;
+
+import org.wso2.carbon.identity.workflow.engine.dao.WorkflowEventRequestDAO;
+import org.wso2.carbon.identity.workflow.engine.dao.impl.WorkflowEventRequestDAOImpl;
+import org.wso2.carbon.identity.workflow.engine.exception.WorkflowEngineException;
+import org.wso2.carbon.identity.workflow.engine.exception.WorkflowEngineRuntimeException;
+import org.wso2.carbon.identity.workflow.engine.util.WorkflowEngineConstants;
+import org.wso2.carbon.identity.workflow.mgt.WorkflowExecutorManagerService;
+import org.wso2.carbon.identity.workflow.mgt.WorkflowExecutorManagerServiceImpl;
+import org.wso2.carbon.identity.workflow.mgt.WorkflowManagementService;
+import org.wso2.carbon.identity.workflow.mgt.WorkflowManagementServiceImpl;
+import org.wso2.carbon.identity.workflow.mgt.bean.Parameter;
+import org.wso2.carbon.identity.workflow.mgt.bean.RequestParameter;
+import org.wso2.carbon.identity.workflow.mgt.bean.Workflow;
+import org.wso2.carbon.identity.workflow.mgt.bean.WorkflowAssociation;
+import org.wso2.carbon.identity.workflow.mgt.dto.WorkflowRequest;
+import org.wso2.carbon.identity.workflow.mgt.exception.InternalWorkflowException;
+import org.wso2.carbon.identity.workflow.mgt.exception.WorkflowException;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+public class DefaultWorkflowEventRequestService implements DefaultWorkflowEventRequest {
+
+    public void addApproversOfRequests(WorkflowRequest request, List<Parameter> parameterList) {
```

Comment:
Add method comments to public methods

Reply:
Addresses with this commit https://github.com/wso2-incubator/identity-workflow-simple/pull/1/commits/0f8bc6df9d488542344585725d992c952c1b640b